client/src/templates/Challenges/components/completion-modal-body.tsx

```diff
@@ -77,6 +77,7 @@ export class CompletionModalBody extends PureComponent<
         <div className='completion-challenge-details'>
           <GreenPass
             className='completion-success-icon'
+            data-testid='fcc-completion-success-icon'
```

Comment:
@huyenltnguyen As these were `svg` and `div` I had to use `testId`... I tried using `role=graphics-document` for `svg` as mentioned [here](https://www.w3.org/TR/html-aria/#docconformance)

However, it did not helped..

and where as for the `div` I could not use `text` as the text is on the inner element and we are validating the style on parent element, which forced me to use `testId`

Reply:
Yeah the role `graphics-document` doesn't seem to be available in [aria-query](https://github.com/A11yance/aria-query/tree/main/src/etc/roles/literal).

The GreenPass component has a `<title>` element that we can query using the [ByTitle](https://testing-library.com/docs/queries/bytitle) method. However, I don't have this offhand, but I do remember seeing an issue reported to React Testing Library regarding not being able to query `title` if it is not a direct child of the `svg` element, and this is exactly what we are having in our codebase.

https://github.com/freeCodeCamp/freeCodeCamp/blob/1e26dc1529cb9274c3a3edc2599d39c6cb84f714/client/src/assets/icons/green-pass.tsx#L12-L21

I'm okay with using `data-testid` in both the `svg` and the `div` 🙂